### ISSUE TITLE
How to use a button

### ISSUE THREAD

**Issue by GeneArnold** [neonious/lowjs_esp32_examples#8]
I'm trying to find some very simple examples to start learning how to use my new Neonious board. How would I connect a button and setup the GPIO pin to an input then console.log when the button was pressed.

Thank you

**Comment by neoniousTR** [comment]
Hi!

Regarding electronics basics (how to wire a button), I just searched the Internet and, as an example, found this: https://www.embeddedrelated.com/showarticle/505.php

Regarding accessing GPIO via JavaScript, here are the docs of the GPIO module: https://www.lowjs.org/lowjs_for_esp32/module-gpio.html
BTW: All GPIOs are set to input on default already.

To make it easier: Did you check the preinstalled example program of the neonious one (if you deleted it, do a factory reset). It also outputs via console.log if you press the user button.

Regards

Thomas

**Comment by GeneArnold** [comment]
Thank you Thomas. I'm good on the wiring part since I've been playing around with Arduino for a short amount of time (though that website is a great source of information). 

What I needed was code to see how to interact with a button. Yes, I actually did see that code on the main page after watching this video of yours.
https://www.youtube.com/watch?v=miTX0Lyygdc&t=307s

That helped but I still would love to learn more about the libraries. I'm not a serious developer so just looking at the module code does not help me enough. I'm sure for more seasoned programmers that would be a perfect solution, but for me it's to advanced.

I love this platform and think that you have done an amazing job with low.js and the Neonious microcontroller. I would love to invest more into this platform and use it for all my projects. The small amount of dev work I do does revolve around javascript and node so using your tools is perfect. I've started using Johnny-Five as well but that does not have the same level of portability as your tools. Plus, I'll be honest here, the neonious IDE you somehow are able to put on these chips is simply amazing!

Can you please start up a forum for use to talk and share? We need a lot more simple examples like controlling LEDs with PMW and basic LCD Hello World examples. I'm also trying to better understand what hardware based Node.js libraries are compatible with low.js.

The more I play the more questions I have and want to learn! 

Thank you very much for responding and putting together this wonderful platform!

**Comment by neoniousTR** [comment]
I am planning a forum in this Q4. We just have one large project from a paying customer to fulfill before doing this. I will keep this issue opened till a forum is in place!

**Comment by GeneArnold** [comment]
Excellent to hear about that forum and that you are pulling in some more
paying customers. I'm blown away by how much you have packed into the
neonious. Today I got mine working with a button and sending out REST
calls. Was pretty cool! I hacked together a debouncer for the button and
would love to learn how to handle this better. All good topics for the
forum! I can see another few neonious controllers in my future for sure!
Great tool ;-)
